src/html.rs

```diff
@@ -354,7 +354,7 @@ fn gen_html(
     ctx.insert("branch_enabled", &branch_enabled);
 
     let items = f
-        .lines()
+        .split(b'\n')
```

Comment:
Why this change ?
It isn't the same as `lines`:
https://doc.rust-lang.org/nightly/std/io/trait.BufRead.html#method.lines

Reply:
We should have some tests to cover different kind of eols.

Reply:
It's not quite the same as lines, in that lines just returns an error if it's invalid UTF-8, versus split gives you `[u8]`'s that you can call `from_utf8_lossy` on. 

I have missed that `Lines` discards trailing CR characters at the end of lines (https://doc.rust-lang.org/nightly/src/std/io/mod.rs.html#2791-2810) so if we stick with `split` that'll have to be fixed.